modules/issue_tracker/jsx/editIssue.js

```diff
@@ -0,0 +1,628 @@
+        },
+        render: function () {
+
+            return (
+                <div className="row form-group">
+                    <div className="col-sm-9">
+                        <div className="btn btn-primary"
+                             onClick={this.toggleCollapsed}
+                             data-toggle="collapse"
+                             data-target="#comment-history"
+                        >
+                            Show Comment History
+                        </div>
+                    </div>
+                    <br></br>
+                    <div id="comment-history">
+                        <div className="col-sm-12">
+                            <div dangerouslySetInnerHTML={{__html: this.props.text}}/>
+                        </div>
+                    </div>
+                </div>
+
+            );
+        }
+    }
+);
+
+var IssueEditForm = React.createClass(
+    {
+
+        propTypes: {
+            DataURL: React.PropTypes.string.isRequired,
+            action: React.PropTypes.string.isRequired,
+        },
+
+        getInitialState: function () {
+            return {
+                'Data': [],
+                'formData': {},
+                'submissionResult': null,
+                'errorMessage': null,
+                'isLoaded': false,
+                'loadedData': 0,
+                'isNewIssue': false,
+                'issueID': 0
+            };
+        },
+
+        componentDidMount: function () {
+            this.getDataAndChangeState();
+        },
+
+        render: function () {
+
+            if (!this.state.isLoaded) {
+                if (this.state.error != undefined) {
+                    return (
+                        <div className="alert alert-danger text-center">
+                            <strong>
+                                {this.state.error}
+                            </strong>
+                        </div>
+                    );
+                }
+                return (
+                    <button className="btn-info has-spinner">
+                        loading
+                        <span className="glyphicon glyphicon-refresh glyphicon-refresh-animate"></span>
+                    </button>
+                );
+            }
+
+            var helpText = "A title and assignee are required"; //todo: here fill out the fields that are neccessary.
+            var alertMessage = "";
+            var alertClass = "alert text-center hide";
+            var hasEditPermission = this.state.Data.hasEditPermission || this.state.Data.isOwnIssue;
+
+            var headerText = " ";
+            if (this.state.isNewIssue) {
+                headerText = "Create New Issue";
+            }
+            else {
+                headerText = "Edit Issue #" + this.state.issueData.issueID;
+            }
+
+            var lastUpdateValue = " ";
+            if (this.state.isNewIssue) {
+                lastUpdateValue = "Never!"
```

Comment:
Although semicolons are optional in JS, their usage is highly encouraged, at very least for consistency with other code in this file.

I suggest you run ESLint with --fix option, to catch and fix all small formatting issues and have a standard looking code.